sensu.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.add_dependency "json" if RUBY_VERSION < "1.9"
```

Comment:
also, fwiw, this only affects the build gem depending on the ruby version used to build it, so it's basically meaningless unless you're building the gem on 1.8